tools/gulp/packaging/build-utils.ts

```diff
@@ -0,0 +1,85 @@
+import {writeFileSync, readFileSync} from 'fs-extra';
+import {join} from 'path';
+import {LICENSE_BANNER, MATERIAL_VERSION} from '../constants';
```

Comment:
Small, single-purpose files are just much easier to grok than have a bunch of unrelated stuff in one file.